src/Common.php

```diff
@@ -41,11 +41,13 @@ public function __construct(array $headers, array $config)
             ),
         );
 
-        $handlerStack->remove('http_errors');
-        $handlerStack->unshift(
-            Middleware::httpErrors(new BodySummarizer(self::MAX_HTTP_ERROR_MESSAGE_LENGTH)),
-            'http_errors',
-        );
+        if (interface_exists('GuzzleHttp\BodySummarizer')) {
```

Comment:
`interface_exists` - neni `BodySummarizer` class?

Reply:
Sakra, já to původně testoval na `BodySummarizerInterface` a jak jsem se motal s phpstanem, uteklo mi to.